packages/notifications/discord/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import type { Monitor, Notification } from "@openstatus/db/src/schema";
```

Comment:
@AmoabaKelvin, within #407 we now have better access to the schema types to avoid code dublication and be more consistent. 💪

So with all our schemas we can use the import the following types:

```
type Monitor = z.infer<typeof selectMonitorSchema>;
type InsertMonitor = z.infer<typeof insertMonitorSchema>;
```